src/main/java/uk/gov/hmcts/reform/pip/subscription/management/repository/SubscriptionRepository.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.pip.subscription.management.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.pip.subscription.management.models.Subscription;
+
+import java.util.List;
+
+@Repository
+public interface SubscriptionRepository extends JpaRepository<Subscription, Long> {
+    /**
+     * This JPA interface allows us to specify specific find methods for the database and it should
+     * create the required functionality for us.
+     */
+
+    List<Subscription> findAllById(Long subId); //finds all subs by subID
```

Comment:
would this not be 1 id? each subscription should have its own unique id so if you search for id `123` there should only be 1 subscriptio with `123`

Reply:
Yeah, this was me being silly. Removed now.